### ISSUE TITLE
Changing player speeds causes a high enough speed to noclip through blocks.

### ISSUE THREAD

**Issue by FNCxPro** [PAT457/Reality#8]
If you use the function player.setSpeed(player.getSpeed() + 1); it will cause a high enough speed to noclip through small clusters of blocks. If you jump with that speed, you will be pushed into the air at high speeds, until gravity takes over and the speed is decreased and the player falls slowly.

**Comment by FlareHAX0R** [comment]
Hmmm.... I'm going to ask to do a massive rewrite of stuff soon - possibly, so there might be a fix.

**Comment by PAT457** [comment]
Yeah, I was playing around with this a few weeks ago, i see the results. Its the collision system. it's buggy as fuck. it doesn't require a full rewrite but it needs some fixing.
